extensions/lua/src/lua.rs

```diff
@@ -58,7 +63,14 @@ impl LuaExtension {
             .ok_or_else(|| format!("no asset found matching {:?}", asset_name))?;
 
         let version_dir = format!("lua-language-server-{}", release.version);
-        let binary_path = format!("{version_dir}/bin/lua-language-server");
+        let binary_path = format!(
+            "{version_dir}/bin/lua-language-server{}",
+            match platform {
+                zed::Os::Mac => "",
+                zed::Os::Linux => "",
+                zed::Os::Windows => ".exe",
+            },
+        );
```

Comment:
```suggestion
        let binary_path = format!("{version_dir}/bin/lua-language-server{}", std::env::consts::EXE_SUFFIX);
```

Reply:
`std::env::consts::EXE_SUFFIX` returns `.wasm` and not `.exe` unfortunately

Reply:
Oh right, we are under wasm env. Sorry.

Reply:
No problem. My knowledge of Rust is quite minimal and your change would be a lot nicer!